DebbyMiressa/To-Do-List
# Jest Unit Testing: Test2

##Test One (Add and Delete task)
In this milestone,
I and my partner @Alhajideen were able to achieve the following;

- Add a test for the edit task with a new description text
- Add a test for the check completed functionality
- Add a test for the clear all checked (completed) tasks

We await your feedback 😄
